apps/authentication-portal/src/main/webapp/emailOtp.jsp

```diff
@@ -123,6 +172,18 @@
                                 String authFailureMsg = request.getParameter("authFailureMsg");
                                 if (authFailureMsg != null && "login.fail.message".equals(authFailureMsg)) {
                         %>
+                        <%
+                            if (reCaptchaEnabled) {
+                                String reCaptchaKey = CaptchaUtil.reCaptchaSiteKey();
+                        %>
+                        <div class="g-recaptcha"
+                            data-sitekey="<%=Encode.forHtmlContent(reCaptchaKey)%>"
+                            data-callback="onSubmit"
```

Comment:
we can directly use `submitOtpForm` function here instead of defining and using a separate function, can't we?